semantic-scope-graph/README.md

```diff
@@ -0,0 +1 @@
+# Semantic Scope Graph
```

Comment:
?

Reply:
I'm having trouble recollecting the specifics now, but at some point while working on this I saw an error from `cabal` because the `semantic-scope-graph` package did not define a `README` file, even though `semantic-scope-graph.cabal` lists a `README` as an `extra-source-file`.

Reply:
I removed this README in https://github.com/github/semantic/pull/507/commits/cf76ab285135e531af986ee09c8f8725d7eb55c0 and tested building after `cabal clean` and was able to build successfully without error. I'm not sure what I experienced earlier but it doesn't appear to be a problem now.